scripts/cmake/generic.cmake

```diff
@@ -21,12 +21,12 @@ set(CMAKE_CUDA_FLAGS "${CMAKE_CUDA_FLAGS}  --Werror all-warnings")
 # to the device compiler
 set(CUDA_PROPAGATE_HOST_FLAGS OFF)
 
-# FIXME(haruhi): -std=c++17 has to be set explicitly here, Otherwise, linking
+# FIXME(haruhi): -std=c++20 has to be set explicitly here, Otherwise, linking
 # against torchlibs will raise errors. it seems that the host compilation
 # options are not passed to torchlibs.
```

Comment:
I am still unable to resolve this issue. The host compilation options are not being passed to the Tor libraries. 😢